src/data/reducers/submissions.js

```diff
@@ -1,7 +1,7 @@
 import actions from 'data/actions';
 
 const initialState = {
-  list: {
+  allSubmissions: {
```

Comment:
Isn't this suppose to be submissionListData?

Reply:
This data is an object of submissionListData objects.  I'm honestly trying to remove the word List from this specific field because its data is an object, just in service of a List view.
Thus, `list.allSubmissions` will return an object of submissionListData entries.